src/Curl/Curl.php

```diff
@@ -109,12 +109,12 @@ class Curl
     public $curl_error_message = null;
 
     /**
-     * @var bool Whether an error occured or not
+     * @var bool Whether an error occurred or not
      */
     public $http_error = false;
 
     /**
-     * @var int Contains the status code of the current processed request.
+     * @var integer Contains the status code of the current processed request.
```

Comment:
I prefer short form here.

https://github.com/php-fig/fig-standards/blob/master/proposed/extended-coding-style-guide-meta.md#43-enforcing-short-form-for-all-type-keywords

The PSR is not yet accepted, but it makes sense to use short forms.

Reply:
i can admit this. i will make tomorrow a commit with short form